projects/cfengine/project.yaml

```diff
@@ -0,0 +1,10 @@
+homepage: "https://github.com/cfengine/core"
+main_repo: "https://github.com/cfengine/core"
+language: c++
+primary_contact: ""
+auto_ccs:
+  - "Adam@adalogics.com"
+sanitizers:
+  - address
+  - undefined
+  - memory
```

Comment:
What happens if the fuzzed code fails an `assert()`?

Reply:
`exit`s and `assert`s are fuzz-blockers and need to be rewritten. I suggest starting with the least-intrusive way of doing this, and then gradually improve the fuzzability. A good starting point could be to define a "fuzz"-macro in the makefile that avoids `exit`s and `assert`s.

Reply:
Can you elaborate on what you mean by fuzz-blockers?

We don't want to remove / rewrite asserts, they are quite valuable.

On the other hand, the fuzzer should never fail an assert, if it does that's a bug (given that we are generating appropriate inputs for the context). That's primarily why I ask. If it hits an assert which fails, do we get a bug report?

Reply:
@AdamKorcz I don't think this comment is resolved, I'm still interested to know if we receive bug reports for failed asserts (I hope so).